ament_copyright/bin/ament_copyright

```diff
@@ -1,4 +1,17 @@
 #!/usr/bin/env python3
+# Copyright 2014 Open Source Robotics Foundation, Inc.
```

Comment:
Do we want a newline before this? Otherwise the shebang line kind of "disappears".